Source/WebCore/workers/DedicatedWorkerGlobalScope.cpp

```diff
@@ -97,6 +97,7 @@ ExceptionOr<void> DedicatedWorkerGlobalScope::postMessage(JSC::JSGlobalObject& s
 
 ExceptionOr<void> DedicatedWorkerGlobalScope::importScripts(const FixedVector<String>& urls)
 {
+    thread().workerObjectProxy().reportPendingActivity(true);
```

Comment:
Should we do this in WorkerGlobalScope? - It seems like any worker importing scripts would need to extend its lifetime like this.

Alternatively, maybe we should move WorkerGlobalScope::importScripts down into DedicatedWorkerGlobalScope::importScripts, since no other subclass uses it?